app/src/main/java/com/cornellappdev/transit/ui/viewmodels/HomeViewModel.kt

```diff
@@ -27,7 +28,12 @@ class HomeViewModel @Inject constructor(
 
     val stopFlow = routeRepository.stopFlow
 
-    //Default map location
+    val lastRouteFlow = routeRepository.lastRouteFlow
```

Comment:
Probably good to write spec for these, even if they're literally just like copied from the `routeRepository`, since these specs will be read by UI implementers.